### PR TITLE
Admin products management

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#45498](https://edu-redmine.sun-asterisk.vn/issues/45498)

## WHAT (optional)
- CRUD products

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/views/admin/books/_book.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<tr class="order-details">
+  <td class="order-name">
+    <%= book.id %>
```
Comment (ghost) [inline app/views/admin/books/_book.html.erb]: id ở đây làm gì em ơi

File: app/views/admin/books/_book.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  </td>
+  <td class="order-price">
+    <%= book.num_pages %>
+  </td>
+  <td class="books-name">
+    <% book.book_authors.each do |at| %>
+      <div>
+        <%= at.author.name %>
+      </div>
+    <% end %>
```
Comment (ghost) [inline app/views/admin/books/_book.html.erb]: book.book_authors.pluck(:name).join(" ,")

File: app/views/admin/books/_books.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <th class="quantity-title"><%= t ".publish_year" %></th>
+    <th class="option-title"><%= t ".price" %></th>
+    <th class="option-title"><%= t ".num_pages" %></th>
+    <th class="option-title"><%= t ".authors" %></th>
+    <th class="option-title"><%= t ".option" %></th>
+  </tr>
+  <div class="orders-list">
+    <% @books.each do |b| %>
+      <%= render "book", book: b %>
+    <% end %>
```
Comment (ghost) [inline app/views/admin/books/_books.html.erb]: nghiên cứu partial collection đi em ơi

**Review by ghost — COMMENTED** [review]
cmt

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ghost** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**